### PR TITLE
cloud: add faq about how to import specified columns, and delete outdated files

### PR DESCRIPTION
**Pull request by hfxsd**
### What is changed, added or deleted? (Required)

- Add faq about how to import specified columns
- Delete 5 outdated files about import. Users now can use the Data Import and migration features in TiDB Cloud to do those tasks.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v7.2 (TiDB 7.2 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v7.0 (TiDB 7.0 versions)
- [ ] v6.6 (TiDB 6.6 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-cloud/tidb-cloud-import-local-files.md [hunk 1]
```diff
@@ -74,3 +74,22 @@ Currently, this method supports importing one CSV file for one task into either
...
+
+No. Currently, you can only import all columns of a CSV file into an existing table.
+
+In this case, you can use [LOAD DATA](/sql-statements/sql-statement-load-data.md) to specify the columns to import. For example:
+
+```sql
+CREATE TABLE `url` (
+  `id` int(11) NOT NULL AUTO_INCREMENT,
+  `client` smallint(5) unsigned NOT NULL,
+  `dest` varchar(5),
```
Comment (hfxsd) [inline tidb-cloud/tidb-cloud-import-local-files.md:90]:
```suggestion
  `address` varchar(5),
```

File: tidb-cloud/tidb-cloud-import-local-files.md [hunk 2]
```diff
@@ -74,3 +74,22 @@ Currently, this method supports importing one CSV file for one task into either
...
+
+```sql
+CREATE TABLE `url` (
+  `id` int(11) NOT NULL AUTO_INCREMENT,
+  `client` smallint(5) unsigned NOT NULL,
+  `address` varchar(5),
+  `employee` smallint(5) unsigned NOT NULL,
+  PRIMARY KEY (`id`)
+) ENGINE=InnoDB;
+LOAD DATA LOCAL INFILE 'load.txt' INTO TABLE URL FIELDS TERMINATED BY ',' (client, dest, employee);
```
Comment (hfxsd) [inline tidb-cloud/tidb-cloud-import-local-files.md:94]:
```suggestion
LOAD DATA LOCAL INFILE 'load.txt' INTO TABLE URL FIELDS TERMINATED BY ',' (client, address, employee);
```

File: tidb-cloud/tidb-cloud-import-local-files.md [hunk 3]
```diff
@@ -74,3 +74,22 @@ Currently, this method supports importing one CSV file for one task into either
...
+
+```sql
+CREATE TABLE `url` (
+  `id` int(11) NOT NULL AUTO_INCREMENT,
+  `client` smallint(5) unsigned NOT NULL,
+  `address` varchar(5),
+  `employee` smallint(5) unsigned NOT NULL,
+  PRIMARY KEY (`id`)
+) ENGINE=InnoDB;
+LOAD DATA LOCAL INFILE 'load.txt' INTO TABLE URL FIELDS TERMINATED BY ',' (client, address, employee);
```
Comment (Frank945946) [inline tidb-cloud/tidb-cloud-import-local-files.md:94]:
```suggestion
CREATE TABLE `import_test` (
  `id` int(11) NOT NULL AUTO_INCREMENT,
  `name` varchar(64) NOT NULL,
  `address` varchar(64) NOT NUL,
  PRIMARY KEY (`id`)
) ENGINE=InnoDB;
LOAD DATA LOCAL INFILE 'load.txt' INTO TABLE import_test FIELDS TERMINATED BY ',' (name, address);
```

File: tidb-cloud/tidb-cloud-import-local-files.md [hunk 4]
```diff
@@ -74,3 +74,21 @@ Currently, this method supports importing one CSV file for one task into either
...
+
+No. Currently, you can only import all columns of a CSV file into an existing table.
+
+In this case, you can use the MySQL client to connect your TiDB cluster, and then use [LOAD DATA](/sql-statements/sql-statement-load-data.md) to specify the columns to import. For example:
+
+```sql
+CREATE TABLE `import_test` (
+  `id` int(11) NOT NULL AUTO_INCREMENT,
+  `name` varchar(64) NOT NULL,
+  `address` varchar(64) NOT NUL,
```
Comment (okJiang) [inline tidb-cloud/tidb-cloud-import-local-files.md:90]:
```suggestion
  `address` varchar(64) NOT NULL,
```

**Review by Frank945946 — APPROVED** [review]

File: tidb-cloud/tidb-cloud-import-local-files.md [hunk 5]
```diff
@@ -74,3 +74,21 @@ Currently, this method supports importing one CSV file for one task into either
...
+In this case, you can use the MySQL client to connect your TiDB cluster, and then use [LOAD DATA](/sql-statements/sql-statement-load-data.md) to specify the columns to import. For example:
+
+```sql
+CREATE TABLE `import_test` (
+  `id` int(11) NOT NULL AUTO_INCREMENT,
+  `name` varchar(64) NOT NULL,
+  `address` varchar(64) NOT NULL,
+  PRIMARY KEY (`id`)
+) ENGINE=InnoDB;
+LOAD DATA LOCAL INFILE 'load.txt' INTO TABLE import_test FIELDS TERMINATED BY ',' (name, address);
```
Comment (okJiang) [inline tidb-cloud/tidb-cloud-import-local-files.md:93]: will report an error

```sql
mysql> CREATE TABLE `import_test` (
    ->   `id` int(11) NOT NULL AUTO_INCREMENT,
    ->   `name` varchar(64) NOT NULL,
    ->   `address` varchar(64) NOT NULL,
    ->   PRIMARY KEY (`id`)
    -> ) ENGINE=InnoDB;
Query OK, 0 rows affected (0.47 sec)

mysql> LOAD DATA LOCAL INFILE 'load.txt' INTO TABLE import_test FIELDS TERMINATED BY ',' (name, address);
ERROR 2068 (HY000): LOAD DATA LOCAL INFILE file request rejected due to restrictions on access.
mysql>
```

File: tidb-cloud/tidb-cloud-import-local-files.md [hunk 6]
```diff
@@ -74,3 +74,21 @@ Currently, this method supports importing one CSV file for one task into either
...
 
 10. On the **Import** page, you can click **View** in the **Action** column to check the import task detail.
+
+## FAQ
+
+### Can I only import some specified columns on TiDB Cloud?
+
+No. Currently, you can only import all columns of a CSV file into an existing table.
+
+In this case, you can use the MySQL client to connect your TiDB cluster, and then use [LOAD DATA](/sql-statements/sql-statement-load-data.md) to specify the columns to import. For example:
```
Comment (hfxsd) [inline tidb-cloud/tidb-cloud-import-local-files.md:84]:
```suggestion
In this case, you can use the MySQL client to connect your TiDB cluster, and then use [LOAD DATA](https://docs.pingcap.com/tidb/stable/sql-statement-load-data) to specify the columns to import. For example:
```

File: tidb-cloud/tidb-cloud-import-local-files.md (hunk 5 above)
Comment (okJiang) [reply tidb-cloud/tidb-cloud-import-local-files.md:93]: I am asking serverless team

Comment (okJiang) [reply tidb-cloud/tidb-cloud-import-local-files.md:93]: The problem is according to MySQL Client

> mysql  Ver 14.14 Distrib 5.7.40, for Linux (x86_64) using  EditLine wrapper
success
> mysql  Ver 8.0.26 for macos11 on arm64 (MySQL Community Server - GPL)
failed

If user encounter the failed case, they can add `--local-infile=true` when connect using `mysql`

File: tidb-cloud/tidb-cloud-import-local-files.md [hunk 7]
```diff
@@ -74,3 +74,21 @@ Currently, this method supports importing one CSV file for one task into either
...
+
+```sql
+CREATE TABLE `import_test` (
+  `id` int(11) NOT NULL AUTO_INCREMENT,
+  `name` varchar(64) NOT NULL,
+  `address` varchar(64) NOT NULL,
+  PRIMARY KEY (`id`)
+) ENGINE=InnoDB;
+LOAD DATA LOCAL INFILE 'load.txt' INTO TABLE import_test FIELDS TERMINATED BY ',' (name, address);
+```
```
Comment (hfxsd) [inline tidb-cloud/tidb-cloud-import-local-files.md:94]:
```suggestion
```

If you use MySQL client and encounter the following error "ERROR 2068 (HY000): LOAD DATA LOCAL INFILE file request rejected due to restrictions on access.",  you can add `--local-infile=true` when connecting.
```

Comment (okJiang) [reply tidb-cloud/tidb-cloud-import-local-files.md:94]: why delete ```?

Comment (hfxsd) [reply tidb-cloud/tidb-cloud-import-local-files.md:94]: With ```, the format is messy. Will add it later. Ignore it.

File: tidb-cloud/tidb-cloud-import-local-files.md [hunk 8]
```diff
@@ -74,3 +74,23 @@ Currently, this method supports importing one CSV file for one task into either
...
+
+If you use a MySQL client and encounter the following error "ERROR 2068 (HY000): LOAD DATA LOCAL INFILE file request rejected due to restrictions on access.", you can add `--local-infile=true` when connecting.
```
Comment (hfxsd) [inline tidb-cloud/tidb-cloud-import-local-files.md:96]:
```suggestion
If you use `mysql` and encounter the following error "ERROR 2068 (HY000): LOAD DATA LOCAL INFILE file request rejected due to restrictions on access.", you can add `--local-infile=true` when connecting.
```

**Review by lilin90 — COMMENTED** [review]
I see that multiple files are deleted. Please confirm whether that is expected and add `aliases` for deleted files.

File: tidb-cloud/tidb-cloud-migration-overview.md [hunk 9]
```diff
@@ -1,6 +1,7 @@
 ---
 title: Migration and Import Overview
 summary: Learn an overview of data migration and import scenarios for TiDB Cloud.
+aliases: ['/tidb-cloud/export-data-from-tidb-cloud']
```
Comment (hfxsd) [inline tidb-cloud/tidb-cloud-migration-overview.md:4]:
```suggestion
aliases: ['/tidbcloud/export-data-from-tidb-cloud']
```

File: tidb-cloud/migrate-from-mysql-using-data-migration.md [hunk 10]
```diff
@@ -1,6 +1,7 @@
 ---
 title: Migrate MySQL-Compatible Databases to TiDB Cloud Using Data Migration
 summary: Learn how to migrate data from MySQL-compatible databases hosted in Amazon Aurora MySQL, Amazon Relational Database Service (RDS), Google Cloud SQL for MySQL, or a local MySQL instance to TiDB Cloud using Data Migration.
+aliases: ['/tidb-cloud/migrate-data-into-tidb','/tidb-cloud/migrate-incremental-data-from-mysql']
```
Comment (hfxsd) [inline tidb-cloud/migrate-from-mysql-using-data-migration.md:4]:
```suggestion
aliases: ['/tidbcloud/migrate-data-into-tidb','/tidbcloud/migrate-incremental-data-from-mysql']
```

File: tidb-cloud/import-csv-files.md [hunk 11]
```diff
@@ -1,6 +1,7 @@
 ---
 title: Import CSV Files from Amazon S3 or GCS into TiDB Cloud
 summary: Learn how to import CSV files from Amazon S3 or GCS into TiDB Cloud.
+aliases: ['/tidb-cloud/migrate-from-amazon-s3-or-gcs','/tidb-cloud/migrate-from-aurora-bulk-import']
```
Comment (hfxsd) [inline tidb-cloud/import-csv-files.md:4]:
```suggestion
aliases: ['/tidbcloud/migrate-from-amazon-s3-or-gcs','/tidbcloud/migrate-from-aurora-bulk-import']
```

File: tidb-cloud/import-csv-files.md [hunk 12]
```diff
@@ -1,6 +1,7 @@
 ---
 title: Import CSV Files from Amazon S3 or GCS into TiDB Cloud
 summary: Learn how to import CSV files from Amazon S3 or GCS into TiDB Cloud.
+aliases: ['/tidbcloud/migrate-from-amazon-s3-or-gcs','/tidbcloud/migrate-from-aurora-bulk-import']
```
Comment (hfxsd) [inline tidb-cloud/import-csv-files.md:4]:
```suggestion
aliases: ['/tidbcloud/migrate-from-amazon-s3-or-gcs']
```

File: develop/dev-guide-insert-data.md [hunk 13]
```diff
@@ -246,7 +246,7 @@ The following are the recommended tools for bulk-insert:
 
 <CustomContent platform="tidb-cloud">
 
-- Data import: [Data Import Task](/tidb-cloud/import-sample-data.md) page in the TiDB Cloud console. You can import **Dumpling** exported data, a **CSV** file, or [Migrate Data from Amazon Aurora to TiDB](/tidb-cloud/migrate-from-aurora-bulk-import.md). It also supports reading data from a local disk, Amazon S3 cloud disk, or GCS cloud disk.
+- Data import: [Data Import Task](/tidb-cloud/import-sample-data.md) page in the TiDB Cloud console. You can import **Dumpling** exported data, a **CSV** file, or [Migrate MySQL-Compatible Databases to TiDB Cloud Using Data Migration](/tidb-cloud/migrate-from-mysql-using-data-migration.md). It also supports reading data from a local disk, Amazon S3 cloud disk, or GCS cloud disk.
```
Comment (hfxsd) [inline develop/dev-guide-insert-data.md:249]:
```suggestion
- Data import: [Data Import Task](/tidb-cloud/import-sample-data.md) page in the TiDB Cloud console. You can import **Dumpling** exported data, a **CSV** file, or [Import CSV Files from Amazon S3 or GCS into TiDB Cloud](/tidb-cloud/import-csv-files.md). It also supports reading data from a local disk, Amazon S3 cloud disk, or GCS cloud disk.
```

File: tidb-cloud/release-notes-2021.md [hunk 14]
```diff
@@ -53,7 +53,7 @@ Bug fixes:
 ## October 21, 2021
 
 * Open user registration to personal email accounts
-* Support [importing or migrating from Amazon S3 or GCS to TiDB Cloud](/tidb-cloud/migrate-from-amazon-s3-or-gcs.md)
+* Support [importing or migrating from Amazon S3 or GCS to TiDB Cloud](/tidb-cloud/migrate-from-mysql-using-data-migration.md)
```
Comment (hfxsd) [inline tidb-cloud/release-notes-2021.md:56]:
```suggestion
* Support [importing or migrating from Amazon S3 or GCS to TiDB Cloud](/tidb-cloud/import-csv-files.md)
```

File: tidb-cloud/tidb-cloud-import-local-files.md [hunk 15]
```diff
@@ -74,3 +74,23 @@ Currently, this method supports importing one CSV file for one task into either
...
 
 10. On the **Import** page, you can click **View** in the **Action** column to check the import task detail.
+
+## FAQ
+
+### Can I only import some specified columns on TiDB Cloud?
+
+No. Currently, you can only import all columns of a CSV file into an existing table.
+
+In this case, you can use the MySQL client to connect your TiDB cluster, and then use [LOAD DATA](https://docs.pingcap.com/tidb/stable/sql-statement-load-data) to specify the columns to import. For example:
```
Comment (hfxsd) [inline tidb-cloud/tidb-cloud-import-local-files.md:84]:
```suggestion
### Can I only import some specified columns into TiDB Cloud?

No. Currently, you can only import all columns of a CSV file into an existing table.

To import only some specified columns, you can use the MySQL client to connect your TiDB cluster, and then use [LOAD DATA](https://docs.pingcap.com/tidb/stable/sql-statement-load-data) to specify the columns to import. For example:
```

File: tidb-cloud/tidb-cloud-import-local-files.md [hunk 16]
```diff
@@ -74,3 +74,23 @@ Currently, this method supports importing one CSV file for one task into either
...
 
 10. On the **Import** page, you can click **View** in the **Action** column to check the import task detail.
+
+## FAQ
+
+### Can I only import some specified columns by the Import feature in TiDB Cloud?
+
+No. Currently, you can only import all columns of a CSV file into an existing table when using the Import feature.
+
+To import only some specified columns, you can use the MySQL client to connect your TiDB cluster, and then use [LOAD DATA](https://docs.pingcap.com/tidb/stable/sql-statement-load-data) to specify the columns to import. For example:
```
Comment (qiancai) [inline tidb-cloud/tidb-cloud-import-local-files.md:84]:
```suggestion
To import only some specified columns, you can use the MySQL client to connect your TiDB cluster, and then use [`LOAD DATA`](https://docs.pingcap.com/tidb/stable/sql-statement-load-data) to specify the columns to be imported. For example:
```

File: tidb-cloud/tidb-cloud-import-local-files.md [hunk 17]
```diff
@@ -74,3 +74,23 @@ Currently, this method supports importing one CSV file for one task into either
...
+CREATE TABLE `import_test` (
+  `id` int(11) NOT NULL AUTO_INCREMENT,
+  `name` varchar(64) NOT NULL,
+  `address` varchar(64) NOT NULL,
+  PRIMARY KEY (`id`)
+) ENGINE=InnoDB;
+LOAD DATA LOCAL INFILE 'load.txt' INTO TABLE import_test FIELDS TERMINATED BY ',' (name, address);
+```
+
+If you use `mysql` and encounter the following error "ERROR 2068 (HY000): LOAD DATA LOCAL INFILE file request rejected due to restrictions on access.", you can add `--local-infile=true` when connecting.
```
Comment (qiancai) [inline tidb-cloud/tidb-cloud-import-local-files.md:96]:
```suggestion
If you use `mysql` and encounter `ERROR 2068 (HY000): LOAD DATA LOCAL INFILE file request rejected due to restrictions on access.`, you can add `--local-infile=true` in the connection string.
```

File: tidb-cloud/troubleshoot-import-access-denied-error.md [hunk 18]
```diff
@@ -11,7 +11,56 @@ After you click **Next** on the **Data Import** page of the TiDB Cloud console a
...
+```
+
+In the sample trust entity:
+
+- `380838443567` is the TiDB Cloud Account ID. Make sure that this field in your trust entity matches your TiDB Cloud Account ID.
+- `696e6672612d617069a79c22fa5740944bf8bb32e4a0c4e3fe` is the TiDB Cloud External ID. Make sure that this field in your trusted entity matches your TiDB Cloud External ID.
+
+### Check whether the role exists
+
+If the role does not exist, create a role following instructions in [Configure Amazon S3 access](/tidb-cloud/config-s3-and-gcs-access.md#configure-amazon-s3-access).
```
Comment (qiancai) [inline tidb-cloud/troubleshoot-import-access-denied-error.md:53]:
```suggestion
If the IAM role does not exist, create a role following instructions in [Configure Amazon S3 access](/tidb-cloud/config-s3-and-gcs-access.md#configure-amazon-s3-access).
```

File: tidb-cloud/troubleshoot-import-access-denied-error.md [hunk 19]
```diff
@@ -11,7 +11,56 @@ After you click **Next** on the **Data Import** page of the TiDB Cloud console a
...
+
+### Check whether the external ID is set correctly
+
+Cannot assume the provided role '{role_arn}'. Check the trust relationships settings on the role. For example, check whether the trust entity has been set to the 'TiDB Cloud account ID' and whether the 'TiDB Cloud External ID' is correctly set in the trust condition. See [Check the trust entity](#check-the-trust-entity).
```
Comment (qiancai) [inline tidb-cloud/troubleshoot-import-access-denied-error.md:57]:
```suggestion
Cannot assume the provided role `{role_arn}`. Check the trust relationships settings on the role. For example, check whether the trust entity has been set to the `TiDB Cloud account ID` and whether the `TiDB Cloud External ID` is correctly set in the trust condition. See [Check the trust entity](#check-the-trust-entity).
```

File: tidb-cloud/changefeed-sink-to-tidb-cloud.md [hunk 20]
```diff
@@ -43,7 +43,7 @@ Before creating a changefeed, you need to export existing data from the source T
     SET GLOBAL tidb_gc_life_time = '720h';
     ```
 
-2. [Export data](/tidb-cloud/export-data-from-tidb-cloud.md) from your TiDB Dedicated cluster, then use community tools such as [mydumper/myloader](https://centminmod.com/mydumper.html) to load data to the destination TiDB Serverless cluster.
+2. [Back up data](/tidb-cloud/backup-and-restore.md#backup) from your TiDB Dedicated cluster, then use community tools such as [mydumper/myloader](https://centminmod.com/mydumper.html) to load data to the destination TiDB Serverless cluster.
```
Comment (qiancai) [inline tidb-cloud/changefeed-sink-to-tidb-cloud.md:46]: 是需要先将数据备份到一个新的集群吗？

File: tidb-cloud/troubleshoot-import-access-denied-error.md [hunk 21]
```diff
@@ -11,7 +11,56 @@ After you click **Next** on the **Data Import** page of the TiDB Cloud console a
 
 To troubleshoot the access denied errors, perform the following checks in the AWS Management Console.
 
-## Check the policy of the IAM user
+## Cannot assume an IAM role
```
Comment (Frank945946) [inline tidb-cloud/troubleshoot-import-access-denied-error.md:14]:
```suggestion
## Cannot assume the provided role 
```

Comment (Frank945946) [reply tidb-cloud/troubleshoot-import-access-denied-error.md:14]: 直接使用报错文案的内容作为标题，方便用户搜索

File: tidb-cloud/troubleshoot-import-access-denied-error.md [hunk 22]
```diff
@@ -11,7 +11,56 @@ After you click **Next** on the **Data Import** page of the TiDB Cloud console a
...
+
+This section describes how to troubleshoot the issue that AWS cannot assume an IAM role.
```
Comment (Frank945946) [inline tidb-cloud/troubleshoot-import-access-denied-error.md:16]:
```suggestion
This section describes how to troubleshoot the issue that TiDB Cloud cannot assume the provided role to access the specified bucket. 
```

File: tidb-cloud/changefeed-sink-to-tidb-cloud.md (hunk 20 above)
Comment (Frank945946) [reply tidb-cloud/changefeed-sink-to-tidb-cloud.md:46]: 是将上游的存量数据导出来，再用 Lightning 等工具导入到 TiDB Cluster

Comment (qiancai) [reply tidb-cloud/changefeed-sink-to-tidb-cloud.md:46]: TiDB Cloud console 上的 backup 功能 https://docs.pingcap.com/tidbcloud/backup-and-restore 支持将存量数据导出来吗

Comment (Frank945946) [reply tidb-cloud/changefeed-sink-to-tidb-cloud.md:46]: 支持的，全量备份就是这么用的

**Review by qiancai — APPROVED** [review]

**Review by Frank945946 — APPROVED** [review]

**Pull request merged**